pkg/services/ngalert/api/api_provisioning.go

```diff
@@ -180,3 +183,39 @@ func (srv *ProvisioningSrv) RouteGetMuteTimings(c *models.ReqContext) response.R
 	}
 	return response.JSON(http.StatusOK, timings)
 }
+
+func (srv *ProvisioningSrv) RoutePostMuteTiming(c *models.ReqContext, mt apimodels.MuteTimeInterval) response.Response {
+	created, err := srv.muteTimings.CreateMuteTiming(c.Req.Context(), mt, c.OrgId)
+	if err != nil {
+		if errors.Is(err, provisioning.ErrValidation) {
+			return ErrResp(http.StatusBadRequest, err, "")
+		}
+		return ErrResp(http.StatusInternalServerError, err, "")
+	}
+	return response.JSON(http.StatusCreated, created)
+}
+
+func (srv *ProvisioningSrv) RoutePutMuteTiming(c *models.ReqContext, mt apimodels.MuteTimeInterval) response.Response {
+	name := web.Params(c.Req)[":name"]
```

Comment:
Not currently - but I created one. Did the same for all path parameters in the provisioning API used it in all these routes. Thanks for pointing this out.

It's a little awkward because the param needs to match the string in the `swagger:route` comment -  and this can't use the constant because it is a comment. This still feels like an improvement, though.